packages/palette/src/elements/Tooltip/Tooltip.tsx

```diff
@@ -87,12 +97,9 @@ const truncate = (tip: string): string => {
   return substring
 }
 
-const Tip = styled(Box)`
-  position: absolute;
-  z-index: 1;
-  transition: opacity 250ms ease-out;
-  text-align: left;
-  box-shadow: ${DROP_SHADOW};
-  cursor: default;
-  pointer-events: none;
-`
+const compose = (a?: (...args: any) => any, b?: (...args: any) => any) => {
+  return (...args) => {
+    a?.(...args)
+    b?.(...args)
+  }
+}
```

Comment:
We have `lodash` in the project

Reply:
We don't :)

Reply:
🤯 wow, impressed... A first?

Reply:
https://github.com/artsy/palette/pull/755

Reply:
haha @dzucconi, of course 👌